master/master.cfg

```diff
@@ -66,8 +66,10 @@ c = BuildmasterConfig = {}
 password = Path('halide_bb_pass.txt').read_text().strip()
 
 _WORKERS = [
+    ('linux-worker-1', 8),
```

Comment:
max builds of 8 seems like a lot. Wouldn't 1 be safer while getting these set up?

Reply:
I don't think so, since the initial tests will be either the nightlies, or me firing off force-builds.

Reply:
I just can't imagine any situation where doing 8 builds at once is safe. The linking step for llvm is liable to use many gigabytes of ram, and if those all occur at the same time...

Reply:
Fair. I'll revisit it.